proposals/0409-access-level-on-imports.md

```diff
@@ -227,6 +230,15 @@ After replacing with an internal import, the transitive dependency loading requi
 but will change for non-resilient modules where transitive dependencies must always be loaded.
 In all cases, updating modules relying on `@_implementationOnly` to instead use internal imports is strongly encouraged.
 
+The scoped imports feature remains independent from the access level declared on the same import.
+Given the example below, the module `Foo` is a public dependency at the module-level and can be referenced from public declaration signatures in the local source file.
+The scoped part, `struct Foo.Bar`, is a hint to the name lookup logic to prioritize resolving references to `Bar` as this one compared to other `Bar` from other imports.
```

Comment:
@beccadax Could you confirm if I describe the right behavior for existing scoped imports?

Reply:
I refined this description in the next commit following our discussion.